src/main/helpers.ts

```diff
@@ -154,19 +150,23 @@ export function onNavigation({ urlTarget, currentUrl, preventDefault, createNewW
   }
 
   const targetWindow = BrowserWindow.getAllWindows().find(b => {
+    const portPathSegment = targetUrl.port ? `:${targetUrl.port}` : ''
     const path = b.webContents.getURL()
-      .replace(`${targetUrl.protocol}//${targetUrl.hostname}`, '');
+      .replace(`${targetUrl.protocol}//${targetUrl.hostname}${portPathSegment}`, '');
```

Comment:
Instead of this, I believe you can just replace `targetUrl.hostname` with `targetUrl.host` and it will include the port